### PR TITLE
[crater only] Stabilize never type, always falling back to !

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
This is likely to create a good amount of regressions, but we want to evaluate just how many to gauge the right path toward stabilization.

r? @ghost

### CODE REVIEW

File: library/core/src/clone.rs [hunk 1]
```diff
@@ -200,7 +200,7 @@ mod impls {
         bool char
     }
 
-    #[unstable(feature = "never_type", issue = "35121")]
+    #[stable(feature = "never_type", since = "1.57.0")]
```
Comment (bugadani) [inline library/core/src/clone.rs:203]: So I admit I don't know anything, but hasn't this ship (i.e. the 1.57 release) sailed already?

**Comment by Mark-Simulacrum** [comment]
@craterbot run mode=check-only cap-lints=warn

**Comment by apiraino** [comment]
Switching labels (hope it's ok) to signal that this does not yet needs review

@rustbot label -S-waiting-on-review +S-experimental

**Comment by faern** [comment]
What's the current status of the never type stabilization? Is there any tracking issue or similar? I'm not able to find any news after this crater run.

**Comment by c410-f3r** [comment]
Zulip channel (https://rust-lang.zulipchat.com/#narrow/stream/259160-t-lang.2Fproject-never-type) is also stalled.

**Pull request closed without merging**